apps/demo/src/locales/en/users.json

```
@@ -1,10 +1,13 @@
 {
   "table": {
     "actions": {
-      "invite": "Invite user"
+      "invite": "Invite user",
+      "enableUser": "Enable User",
+      "disableUser": "Disable User"
```

Lets be consistent with naming here. Since we have "invite" (instead of "inviteUser"), use "enable" and "disable" as the keys. And use sentence case for the message.
```suggestion
      "enable": "Enable user",
      "disable": "Disable user"
```